tpp-app/App.js

```diff
@@ -96,15 +108,48 @@ export function MyTabs() {
 
 
 export default function App() {
+  const navigationRef = useNavigationContainerRef();
+
+  // Requests for notification permissions and also creates a local notification listener
+  // Does not handle remote notifications from a server.
+  useEffect(() => {
+
+    PushNotificationIOS.addEventListener('localNotification', ((notification) => {
```

Comment:
isn't the event listener here identical to the previously defined `onLocalNotification`? If it is you should just replace it with the function u defined earlier